src/app/shared/components/display/modal/modal.component.html

```diff
@@ -0,0 +1,9 @@
+<div class="invite-card">
+  <h2 *ngIf="title">{{ title }}</h2>
+  <p *ngIf="paragraph">{{ paragraph }}</p>
+
+  <div class="invite-actions">
+    <app-button [type]="'text'" [label]="copyButtonText"></app-button>
```

Comment:
Esse botão secundário seria renderizado apenas se for passado algo no input do componente. (A explicação está no arquivo do componente modal)

```ts
<app-button *ngIf="buttonSecondary" [type]="'text'" [label]="buttonSecondary.text" (click)="buttonSecondary.onClick"></app-button>